src/yardstick/capture.py

```diff
@@ -111,7 +125,8 @@ def one(
     return scan_config
 
 
-def result_set(  # pylint: disable=too-many-locals
+# pylint: disable=too-many-branches, too-many-locals
```

Comment:
This seems like signal that it might be time to refactor.

Reply:
since I don't have the time now to do that, I'll remove the log lines